src/prompts/adapter.ts

```diff
@@ -19,9 +19,7 @@ export default class CodeAdapter implements IPrompter {
     constructor(
         private vscodeWrapper: VscodeWrapper
     ) {
-        // TODO Decide whether output channel logging should be saved here?
-        this.outChannel = this.vscodeWrapper.createOutputChannel(Constants.outputChannelName);
-        // this.outChannel.clear();
+        this.outChannel = this.vscodeWrapper.outputChannel;
```

Comment:
I'm not understanding how this is removing any channels. Both this and the vscodeWrapper were using the same channel name - MSSQL

Reply:
Oh this is just getting rid of the duplicate MSSQL one. Ok, that's fine. I'm not sure what the standard is for extensions contributing channels, but 3 seems like a lot for a single extension. The SqlToolsService initialization errors in particular seem like something that would be better to just have in a log file or something instead - that's rarely going to be something users need/want to check.

Reply:
It's always been like this, and I don't want to make a change unless we got a customer issue about it since the users are probably used to it. The duplicate `MSSQL` was clearly a bug however.